target/linux/ath79/dts/qca9533_dlink_dap-1330-a1.dts

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "qca9533_dlink_dap-13xx.dtsi"
+
+/ {
+	compatible = "dlink,dap-1330-a1", "qca,qca9533";
+	model = "D-Link DAP-1330-A1";
```

Comment:
No hyphen before A1?

Reply:
It's amazing how I could read 10x through the whole diff, and still there's always these kinds of things left...

Reply:
What's even more amazing is how @adrianschmutzler picks these things up without error. What a wizard! 😂